### ISSUE TITLE
Allow to override aria-label for full screen button and thumbnails

### ISSUE THREAD

**Issue by leandrocrs** [xiaolin/react-image-gallery#670]
**Is your feature request related to a problem? Please describe.**
We are using this component in a Brazilian Portuguese application, where some users lay on screen readers to navigate through application, and for our surprise we cannot override the Fullscreen button and thumbnails aria-labels to match our language.

**Describe the solution you'd like**
Allow to override the aria-label through props, os even better, allow to include whatever button props we want, like some frameworks does. Material-ui for example (see `inputProps` in the [Input API Docs page](https://mui.com/material-ui/api/input/#main-content).

```tsx
  <ReactImageGallery fullscreenButtonProps={{ 'aria-label': 'My custom aria-label' }} // .... />
```

**Describe alternatives you've considered**
If allowing overwrite whatever prop is not intended for this moment, at last `aria-label`, maybe `fullscreenButtonAriaLabel` or `fullscreenButtonProps={{ 'aria-label': 'My custom aria-label' }}`.

For image thumbnail, would be a new prop inside `ReactImageGalleryItem`.
```tsx
{
  // ...
 thumbnailAriaLabel: 'Ir para o slide 1',
  // ...
}
```

**Additional context**
N/A.

**Comment by xiaolin** [comment]
You're able to override the fullscreen button via `renderFullscreenButton`. 

Happy to support `thumbnailAriaLabel` for thumbnails, feel free to open PR.
